app/helpers/application_helper.rb

```diff
@@ -1,2 +1,15 @@
 module ApplicationHelper
+  #for nice notifications with toastr
+  def custom_bootstrap_flash
+    flash_messages = []
+    flash.each do |type, message|
+      type = 'success' if type == 'notice'
+      type = 'error'   if type == 'alert'
+      puts type
+
+      text = "<script>toastr.#{type}('#{message}');</script>"
```

Comment:
Nope, that's actually the opposite! 

`html_safe` marks the string as safe, which it might not be!